bootstrap/manifest_template.json

```diff
@@ -125,12 +125,72 @@
     }
   },
   {
+    "id": "redis",
+    "app": {
+      "name": "redis",
+      "meta": {"flynn-system-app": "true"}
+    },
+    "action": "run-app",
```

Comment:
You should be able to use `deploy-app` and deploy Redis after `controller` is up and running. This will avoid the 2 stage process of `run-app` followed by `deploy-app`.